middleware/withCors.ts

```diff
@@ -18,8 +21,16 @@ function runMiddleware(req, res, fn) {
   })
 }
 
+function runLogger(req) {
+  const {method, url, ip, ua, body} = {...req} as any; // eslint-disable-line
+  const pathname = url.split('/api')[1];
+  if (!ignorePaths.some(k => pathname.includes(k)))
+    info({method, ip, ua, pathname, body});
+}
+
 const withCors = (handler) => {
   return async (req, res) => {
+    runLogger(req);
```

Comment:
by using the logger here, we fail to catch errors from the crash of the normal api, by runing it on the middleware we gain the `catch` ability - along with any `throw Error()` being correctly parsed :)

Reply:
@clarkjoao I just noticed that all that is needed is on line 38 we need to log the error that was sent via `error(e)`